source/main/utils/setupLogging.js

```diff
@@ -25,7 +25,9 @@ export const setupLogging = () => {
   log.transports.console.level = isTest ? 'error' : 'info';
   log.transports.rendererConsole.level = isDev ? 'info' : 'error';
   log.transports.file.level = 'debug';
-  log.transports.file.maxSize = 20 * 1024 * 1024;
+  log.transports.file.maxSize = 5 * 1024 * 1024;
+  log.transports.file.maxItems = 4;
```

Comment:
I don't understand why, but setting `maxItems = 4` is actually leaving only 2 old logs. Setting it 5 will leave the correct 3 logs. For a quick test, I set `log.transports.file.maxSize = 100;`

Reply:
@daniloprates I think that @tomothespian saw similar result when he set the maxSize to a similarly small limit as you did. Please try to set the limit to 1MB. Seems that rotation happens to quickly when we set the limit very low.
Then again, perhaps this points into a bug in the logger code.
@yakovkaravelov can you please investigate this?
We need an explanation of this behavior at least 🙏

Reply:
@daniloprates 
Here, `maxItems = 4` means there are 4 logs including 3 old logs and 1 active log.
Above you are reporting it is leaving only 2 old logs, did you wait until active log exceeds the limit?
if you wait until then, there will be 1 more old file generated, and there will be 3 old logs exactly.
Please test again.